crates/e2e/src/testing_node.rs

```diff
@@ -298,7 +299,22 @@ impl TestingNode {
             .expect("failed to get last block number from database");
         self.last_db_block_on_stop = Some(last_db_block);
 
-        execution_node.shutdown().await
+        execution_node.shutdown().await;
+
+        // Acquire a RW transaction and immediately drop it. This blocks until any
```

Comment:
Is this something that needs to be fixed upstream in reth? It seems like the graceful shutdown isn't quite as graceful then.

Or is this a peculiarity of how we are using reth in tests?